packages/frontend/.eslintrc.json

```diff
@@ -1,6 +1,12 @@
 {
   "extends": ["next/core-web-vitals", "plugin:prettier/recommended"],
   "rules": {
-    "no-unused-vars": "error"
+    "no-unused-vars": "error",
+    "prettier/prettier": [
+      "warn",
+      {
+        "endOfLine": "auto"
+      }
+    ]
```

Comment:
Ok, I see you put some comments in the commits :D It's a bit better if you add the comments here (easy to review!)

Reply:
its better if its warn, as it'll autocorrect without interfering with users.  I think all eslint, prettier autcorrect should be warns as that stops them from interfering with the user (but the rules are still enforced).